lib/components/nav_bar.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+
+class NavBar extends StatelessWidget {
+  final int pageIndex;
+  final Function(int) onTap;
+
+  const NavBar({
+    super.key,
+    required this.pageIndex,
+    required this.onTap,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: EdgeInsets.only(
+        left: 16,
```

Comment:
Warum brauchen wir links und rechts nen Margin? Das sieht komisch aus im letztendlichen Screen. Siehe comment in der pr #1